homeassistant/components/automation/__init__.py

```diff
@@ -138,7 +142,7 @@ def is_on(hass, entity_id):
             tasks.append(
                 entity.async_trigger(
                     service_call.data.get(ATTR_VARIABLES),
-                    skip_condition=True,
+                    skip_condition=service_call.data.get(CONF_SKIP_CONDITION),
```

Comment:
Since there is a default value, I believe you want `service_call.data[CONF_SKIP_CONDITION]`. 

**Edit:** I think the same applies for the line above (although you didn't change it in this pull request).

Reply:
Thanks. You're right

Reply:
@JeffLlrion please approve this PR if you're happy with it. This will move the PR into the reviewers approved category.